common/main/ta.xml

```diff
@@ -11105,12 +11105,12 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 	</typographicNames>
 	<personNames>
 		<nameOrderLocales order="givenFirst">und</nameOrderLocales>
-		<nameOrderLocales order="surnameFirst">ko ta te vi yue zh</nameOrderLocales>
+		<nameOrderLocales order="surnameFirst">ko si ta te vi yue zh</nameOrderLocales>
 		<foreignSpaceReplacement xml:space="preserve"> </foreignSpaceReplacement>
 		<initialPattern type="initial">↑↑↑</initialPattern>
 		<initialPattern type="initialSequence">↑↑↑</initialPattern>
 		<personName order="givenFirst" length="long" usage="referring" formality="formal">
-			<namePattern>{given} {given2} {surname} {surname2} {suffix}</namePattern>
+			<namePattern>{prefix} {given} {given2} {surname} {surname2} {suffix}</namePattern>
```

Comment:
This change to add `{prefix}` should be in lines 11166-11168 for `<personName order="surnameFirst" length="long" usage="referring" formality="formal"`

Reply:
I just retracted this change, and added the {prefix} further down